cmp/cmpopts/errors_xerrors.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021, The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !go1.13
```

Comment:
Add a TODO in this file to drop it when the minimally supported version of `cmp` is greater than or equal to Go 1.13.

Reply:
Something like:

// TODO(≥go1.13): For support on <go1.13, we use the xerrors package.
// Drop this file when we no longer support older Go versions.